### DISCUSSION TITLE
Can't find the new toggle button

### DISCUSSION THREAD

**Discussion by Dumonster** [JurajNyiri/HomeAssistant-Tapo-Control#749] [General]
Sorry. I got so excited by the update that I forgot to read where it was and can't seem to find the patch notes.

Thanks for all the hard work!

**Comment by JurajNyiri** [comment]
It’s described in:
- release 6.0.0
- My update in issue
- Integration when asking to input cloud password
- FAQ in readme 

link: https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/releases/tag/6.0.0

**Reply by Dumonster** [reply]
Thanks for your reply. The button isn't there. I guess the next app update will have it.
